hummingbird/ml/operator_converters/onnx/feature_vectorizer.py

```diff
@@ -0,0 +1,33 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+Converter for ONNX-ML Array Feature Extractor.
```

Comment:
I think this should be FeatureVectorizer ?  or is this another name for it?

Reply:
ops. yes.